components/toast.css

```diff
@@ -0,0 +1,36 @@
+.close-btn:hover {
+	color: var(--black);
+}
+
+.toast {
+	width: 100%;
+	max-width: 40rem;
+	border: 1px solid var(--gray);
+}
+
+/* positions */
+.top-left {
+	top: 2rem;

```

Comment:
since you have this top and bottom property repeating a lot in your code you could put them into a common class container. do with everything else that common.

Reply:
It's only for this specific component for ease of the user, elsewhere I have used class where possible.